types/regexp.prototype.flags/regexp.prototype.flags-tests.ts

```diff
@@ -0,0 +1,11 @@
+import flags = require("regexp.prototype.flags");
+import "regexp.prototype.flags/auto";
+
+flags(/foo/) === ""; // $ExpectType boolean
```

Comment:
Hm, did this not work as just Flags without the comparison? Any comparison is going to be boolean, unless I'm missing something...

Reply:
nope, sadly it didn't :-/ it said, basically, "expected foo.Flags, got <the actual full union type foo.Flags represents>" which was the same kind of error i had before.

it's very weird to me that the tests wouldn't either expand `foo.Flags` to the full union, or, realize the full union is the same as `foo.Flags` and pass.

Reply:
Weird, I have no idea why the alias isn't preserved in this case, but I also see that locally.